yaml_files/simphony_metadata.yml

```diff
@@ -418,5 +418,36 @@ CUDS_KEYS:
     definition: Multiphase model
 
   ELECTROSTATIC_MODEL:
-    parent: CUBA.PHYSICS_EQUATION
     definition: Electrostatic model
+    parent: CUBA.PHYSICS_EQUATION
+    models: [CUBA.CFD, CUBA.MOLECULAR_DYNAMICS, CUBA.DEM]
+
+  BASIS:
+    definition: Space basis vectors
+    parent: CUBA.CUDS_COMPONENT
+    CUBA.VECTOR:
+      shape: (3, 3)
+      default: [[1, 0, 0], [0, 1, 0], [0, 0, 1]]
+
+  ORIGIN:
+    definition: The origin of a space system
+    parent: CUBA.CUDS_COMPONENT
+    CUBA.POINT:
+      default: [0, 0, 0]
+      scope: CUBA.USER
+
+  BOX:
+    definition: A simple hexahedron box object
+    parent: CUBA.BOUNDARY
+    CUBA.VECTOR:
+      default: [0, 0, 0]
+      scope: CUBA.USER
+    CUBA.FACE:
+      default: 0
+      scope: CUBA.USER
+    CUBA.BASIS:
+      default: CUBA.BASIS
+      scope: CUBA.USER
+    CUBA.ORIGIN:
```

Comment:
Do we need this here? if we define it then we would need to define a global coordinate system, Since there is no need for it now, how about removing it!